cumulusci/tasks/bulkdata/tests/test_extract.py

```diff
@@ -374,24 +503,112 @@ def test_convert_lookups_to_id__sqlite(self):
 
     @mock.patch("cumulusci.tasks.bulkdata.extract.create_table")
     @mock.patch("cumulusci.tasks.bulkdata.extract.mapper")
-    def test_create_table(self, mapper_mock, create_mock):
-        task = _make_task(
-            ExtractData, {"options": {"database_url": "sqlite:///", "mapping": ""}}
-        )
-        mapping = {
-            "sf_object": "Account",
-            "fields": {"Name": "Name"},
-            "lookups": {},
-            "table": "accounts",
-            "sf_id_table": "test_ids",
-            "oid_as_pk": True,
-        }
-        task.models = {}
-        task.metadata = mock.Mock()
-        task._create_table(mapping)
-        create_mock.assert_called_once_with(mapping, task.metadata)
+    def test_create_table__account_or_contact__person_accounts_disabled(
+        self, mapper_mock, create_mock
+    ):
+        for sf_object, table in [
+            ("Account", "accounts"),
+            ("ACCOUNT", "accounts"),
+            ("account", "accounts"),
+            ("Contact", "contacts"),
+            ("CONTACT", "contacts"),
+            ("contact", "contacts"),
+        ]:
```

Comment:
I'm not a fan of this way of cyling through different parameters for a test, because when it breaks the test output does not make it clear which input failed. A better way to do this is with pytest's parametrization feature: https://docs.pytest.org/en/stable/parametrize.html

Reply:
I refactored the tests.   Also, I was unable to https://docs.pytest.org/en/stable/parametrize.html because `test_load.py` uses `UnitTest.TestCase`, and pytest.mark.paramertize [is not supported](https://docs.pytest.org/en/3.0.1/unittest.html) for `UnitTest.TestCase`.